lib/deaf_grandma.rb

```diff
@@ -19,9 +19,17 @@ def run!
 
 
   def speak(input)
-
-    #Implement your code here <<<<<<<<<
-
+    if(input.eql?("BYE") and @bye_counter == 2)
```

Comment:
ruby styling suggestions: 
 * prefer `&&` to `and`
 * prefer `==` to `#eql?`
 * omit parens around an if-clause, unless they're needed for clarity